terraform-common.mk

```diff
@@ -1,6 +1,7 @@
 ENV_NAME := $(notdir $(shell cd $(PWD) && pwd))
 ENV_SHORT ?= $(word 2,$(subst -, ,$(ENV_NAME)))
 INFRA ?= $(word 1,$(subst -, ,$(ENV_NAME)))
+ENV_TAIL ?= $(subst $(INFRA)-,,$(ENV_NAME))
```

Comment:
That's used in the `trvs` commands as the "environment" since we have a finer-grained separation of terraform infras than simply staging/production.